bn-models/src/models/trips-api/catches.ts

```diff
@@ -64,10 +80,13 @@ export interface Catches extends Base {
 
 interface Haul {
     haulNum?: number;
-    gearTypeCode?: string;
-    gearPerSet?: number;
-    gearLost?: number;
-    avgHooksPerSeg?: number;
+    gear?: gearType;
+    netType?: netType;       // required if gear = 'trawl'
+    codendCapacity?: number; // required if gear = 'trawl'
```

Comment:
believe the fields below are already defined in catch.ts: codendCapacity, isCodendLost, gearPerSet, gearLost, avgHooksPerSeg. Are you pulling the latest version?

Reply:
these aren't new, just moving them and adding comments